cf/field.py

```diff
@@ -10455,9 +10479,9 @@ def convolution_filter(
                     new_bounds[0 : length - lower_offset, 1:] = old_bounds[
                         lower_offset:length, 1:
                     ]
-                    new_bounds[
-                        length - lower_offset : length, 1:
-                    ] = old_bounds[length - 1, 1:]
+                    new_bounds[length - lower_offset : length, 1:] = (
```

Comment:
This was just from linting the files I touched in this PR. For the release we can re-lint the whole codebase since it seems we are a bit behind on linting (`black` edits >10 files under `cf/`).